packages/cdk8s-plus/.projenrc.js

```diff
@@ -48,4 +48,10 @@ project.addScripts({
   build: 'jsii --silence-warnings=reserved-word && yarn docgen'
 });
 
+// this has to be done so because otherwise its gets overridden
+// by the jsii-release version configured in projen.
+project.addDevDependencies({
+  'jsii-release': Semver.caret('0.1.8'),
```

Comment:
jsii-release is only needed in the root package.json, why do we need it here as well?

Reply:
yeah you're right. I'll change it.

Reply:
@eladb Done